docs.feldera.com/docs/pipelines/latency.md

```diff
@@ -0,0 +1,211 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Measuring Pipeline Latency
+
+Incremental view maintenance enables Feldera to quickly incorporate changes to input data into analytical queries.
+In practice it is often important to know exactly how much processing delay a Feldera pipeline introduces to your application.
+This article covers the key latency metrics Feldera exposes, and explain how they can help you measure
+and understand the end-to-end processing latency of your pipelines.
+
+## Background: Understanding pipeline latency
+
+A Feldera pipeline processes data in several stages:
+
+* **Input connectors** read data from the data source (e.g., a database table or a Kafka stream) and parse it
+  into a stream of data change events (inserts, deletes, and updates to SQL tables).
+* **Input queues** buffer incoming data until the SQL engine is ready to
```

Comment:
in the diagram the queue is part of the connector, but this text does not say that